etcd/watchmanager/manager.go

```diff
@@ -95,14 +100,15 @@ func (w *manager) Watch(key string) {
 	ticker := time.Tick(w.opts.WatchChanCheckInterval()) //nolint: megacheck
 
 	var (
+		startRev  int64
```

Comment:
nit: can you call this `revOverride` or something like that? it seems a bit confusing to me looking at this var here while the user already passed in a `withRev`